components/popover/stories/popover.stories.js

```diff
@@ -262,3 +312,135 @@ Nested.args = {
 		}),
 	],
 };
+
+/**
+ * By default, popovers do not have a tip. Popovers without a tip should be used when the source has a
+ * visually distinct down state, in order to show the connection between the popover and its source.
+ * 
+ * This example uses the Menu component within the popover.
+ */
+export const Standard = Template.bind({});
+Standard.storyName = "Default";
+Standard.args = {
+	isOpen: true,
+	trigger: () => null,
+};
+Standard.tags = ["!dev"];
```

Comment:
Since autodocs is included by default, I think it makes sense to not add `autodocs` to every story--- I dug into this again to see _where_ the default is, and it appears that we should make a configuration change that adds the `autodocs` tag to the global settings; it replaced our `autodocs: true` [setting and that setting is going away in Storybook 9.](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mainjs-docsautodocs-is-deprecated) I'll make a PR for that!

Reply:
A PR is up for that: #2924